moveit_core/constraint_samplers/src/constraint_sampler_manager.cpp

```diff
@@ -116,9 +118,10 @@ constraint_samplers::ConstraintSamplerPtr constraint_samplers::ConstraintSampler
       JointConstraintSamplerPtr sampler(new JointConstraintSampler(scene, jmg->getName()));
       if (sampler->configure(jc))
       {
-        logDebug("Temporary sampler satisfying joint constraints for group '%s' allocated. Looking for different types "
-                 "of constraints before returning though.",
-                 jmg->getName().c_str());
+        CONSOLE_BRIDGE_logDebug("Temporary sampler satisfying joint constraints for group '%s' allocated. Looking for "
+                                "different types "
```

Comment:
clang formatting cannot nicely handle line breaks in literals.
@pholthau May I ask you to manually adjust weird line breaks like these?